doc/examples/reconst_cti.py

```diff
@@ -0,0 +1,152 @@
+	
+	where K_{\text{intra}} refers to intra-compartmental kurtosis, 
+	K_{\text{T}} refers to total kurtosis, 
+	K_{\text{aniso}} refers to anisotropic source of kurtosis, 
+	K_{\text{iso}} refers to isotropic source of kurtosis
+
+In the following example we show how to fit the correlation tensor model on a real life dataset and how to estimate correlation tensor based statistics. 
+
+First, we'll import all relevant modules. 
+""" 
+import matplotlib.pyplot as plt
+import numpy as np
+import math
+
+from dipy.reconst.tests.test_qti import _anisotropic_DTD, _isotropic_DTD
+from dipy.core.sphere import Sphere, HemiSphere
+from dipy.reconst.tests.test_qti import _anisotropic_DTD, _isotropic_DTD
+import dipy.reconst.qti as qti
+from dipy.reconst.qti import (from_3x3_to_6x1)
+from dipy.reconst.dti import (decompose_tensor, mean_diffusivity)
+import dipy.reconst.cti as cti
+from dipy.io import read_bvals_bvecs
+from dipy.core.gradients import gradient_table
+from dipy.io.image import load_nifti
+"""
+CTI analysis requires specific gradient information from the data, often acquired from different gradient tables. In this example, we manually load b-values and b-vectors from two separate sets of files, 'bvals1.bval' and 'bvec1.bvec', and 'temp_data/bvals2.bval' and 'temp_data/bvec2.bvec'. 
+The users should ensure that the data is formatted correctly for the CTI analysis they are performing.
+""" 
+data, affine = load_nifti('temp_data/RB_invivo_cti_data_f3.nii')
+bvals1, bvecs1 = read_bvals_bvecs('temp_data/bvals1.bval',
+                                  'temp_data/bvec1.bvec')
+bvals2, bvecs2 = read_bvals_bvecs('temp_data/bvals2.bval',
+                                  'temp_data/bvec2.bvec')
+                                  """ 
+In this example, the function `load_nifti` is used to load the CTI data from the file 'RB_invivo_cti_data_f3.nii' and returns the data as a nibabel Nifti1Image object along with the affine transformation. The b-values and b-vectors for two different gradient tables are loaded from 'bvals1.bval' and 'bvec1.bvec', and 'bvals2.bval' and 'bvec2.bvec' respectively using the `read_bvals_bvecs` function. These are then  further converted into gradient tables `gtab1` and `gtab2` which are essential for the CTI analysis being conducted.
+"""
+gtab1 = gradient_table(bvals1, bvecs1)
+gtab2 = gradient_table(bvals2, bvecs2)
+"""
+Before fitting the data, we perform some data pre-processing. We first compute a brain mask to avoid unnecessary calculations on the background of the image.
+"""
+maskdata, mask = median_otsu(data, vol_idx=[0, 1], median_radius=4, numpass=2,
+                             autocrop=False, dilate=1)
+"""
+Now that we have loaded and created a mask for the data we can go forward with CTI fitting. For this, the CTI model is first defined for the data’s GradientTable object by instantiating the CorrelationTensorModel object in the following way:
```

Comment:
Now that we loaded the data and generated the two gradient tables we can go forward with CTI fitting. For this, the CTI model is first defined for GradientTable objects gtab1 and gtab2 by instantiating the CorrelationTensorModel object in the following way: